src/libs/actions/IOU.ts

```diff
@@ -15178,6 +15178,181 @@ function addReportApprover(
     API.write(WRITE_COMMANDS.ADD_REPORT_APPROVER, params, onyxData);
 }
 
+function updateMultipleMoneyRequests(
+    transactionChangesMap: Record<string, TransactionChanges>,
+    policy: OnyxEntry<OnyxTypes.Policy>,
+    reports: OnyxCollection<OnyxTypes.Report>,
+    transactions: OnyxCollection<OnyxTypes.Transaction>,
+) {
+    for (const [transactionID, transactionChanges] of Object.entries(transactionChangesMap)) {
+        const transaction = transactions?.[`${ONYXKEYS.COLLECTION.TRANSACTION}${transactionID}`];
+        if (!transaction) {
+            continue;
+        }
+
+        const transactionThreadReportID = transaction.reportID;
+        const transactionThread = reports?.[`${ONYXKEYS.COLLECTION.REPORT}${transactionThreadReportID}`] ?? null;
+        const iouReport = reports?.[`${ONYXKEYS.COLLECTION.REPORT}${transactionThread?.parentReportID}`] ?? null;
+        const isFromExpenseReport = isExpenseReport(iouReport);
+
+        const updates: Record<string, string | number> = {};
+        if (transactionChanges.merchant) {
```

Comment:
Should we still check to see if the field is editable for each transaction before adding it to the updates object?